app/helpers/application_helper/button/physical_server_feature_button.rb

```diff
@@ -0,0 +1,9 @@
+class ApplicationHelper::Button::PhysicalServerFeatureButton < ApplicationHelper::Button::GenericFeatureButton
```

Comment:
why do we need this class?

and if we need it why do we override the `visible?` behavior (as that is really what the `GenericFeatureButton` is about).